data/wacom.example

```diff
@@ -49,6 +49,39 @@ Height=6
 # even if the relevant kernel flags would state otherwise.
 IntegratedIn=Display;System
 
+# EDID information if IntegratedIn is Display or System.
+# These are the 8 bytes in the EDID 1.4 struct comprising manufacturer ID,
+# product code and serial number. For reference, edid-decode output:
+# Extracted contents:
+# header:          00 ff ff ff ff ff ff 00
+# serial number:   10 ac 12 ab 4c 56 4a 30 16 16
+#                  ^^^^^ ^^^^^ ^^^^^^^^^^^
+#                   MID   PC     Serial
+#
+# The ProductName is separate in the EDID Other Monitor Descriptors as code
+# 0xFC.
+#
+# A match group is defined as string in the format: <MID>:<PC>:<NAME>:<SERIAL>.
+#
+#  Manufacturer ID: a 3-byte string, e.g. DEL. These are three 5-bit chars from
+#                   the 15 least-significant bits of the first two bytes in the
+#                   EDID struct where 'A' == 1, 'B' == 2, etc. See the EDID
+#                   documentation. Example:
+#                   DEL == (4 << 10) | (5 << 5) | 12 == 0x10ac
+#  Product code: a 4-character hex string in big endian order, e.g. 12ab
```

Comment:
The EDID stores the product code in little-endian order, so I believe "12 ab" from the example hexdump would become "ab12" when converted to big-endian.

Reply:
whoops, I'm gonna switch this to LE instead, no point in using BE here if EDID itself is LE